src/App.tsx

```diff
@@ -1,27 +1,89 @@
-import React from 'react';
-import './App.scss';
+import React, { useState } from 'react';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { Good } from './types/Good';
+
+import 'bulma';
+import './App.scss';
+
+import * as goodsAPI from './api/goods';
+
+export const App: React.FC = () => {
+  const [selectedGoods, setSelectedGoods] = useState<Good[]>([]);
+  const showAllGoods = () => {
+    goodsAPI.getAll().then(goods => setSelectedGoods(goods));
+  };
+
+  const showAllGoodsSortedByColor = () => {
+    goodsAPI.getAllSortedByColor()
+      .then(goods => setSelectedGoods(goods));
+  };
+
+  const show5FirstGoods = () => {
+    goodsAPI.get5First().then(goods => setSelectedGoods(goods));
+  };
+
+  const showRedGoods = () => {
+    goodsAPI.getRedGoods()
+      .then(goods => setSelectedGoods(goods));
+  };
```

Comment:
You have 4 functions that do almost the same but call different callbacks: `getAll()`, `getAllSortedByColor()` etc.
Let's create a one function - `loadGoods`, that will accept a callback as a parameter and just call it